README.md

```diff
@@ -3,18 +3,16 @@
 Retainer is an Android library that helps you retain objects in an Activity / Fragment during configuration changes.
 
 It auto-generates a Fragment internally by using annotation processing.
-A generated Fragment is called `setRetainInstance(true);` in its `onCreate()` to preserve stateful objects.
```

Comment:
I'm afraid "A fragment is called a method" implies "the name of the fragment is the method".